core/src/test/java/com/sequenceiq/cloudbreak/service/upgrade/rds/RdsUpgradeServiceTest.java

```diff
@@ -296,12 +297,6 @@ void testCheckUpgradeRdsOnDatalakeWithStoppedDatahubsAndEntitlementEnabled() {
         verify(stackService, never()).getByWorkspaceId(1L, stack.getEnvironmentCrn(), List.of(StackType.WORKLOAD));
     }
 
-    private StackDatabaseServerResponse createDatabaseServerResponse(MajorVersion majorVersion) {
-        StackDatabaseServerResponse stackDatabaseServerResponse = new StackDatabaseServerResponse();
-        stackDatabaseServerResponse.setMajorVersion(majorVersion);
-        return stackDatabaseServerResponse;
-    }
-
     @ParameterizedTest
     @EnumSource(value = Status.class, names = {"AVAILABLE", "MAINTENANCE_MODE_ENABLED", "EXTERNAL_DATABASE_UPGRADE_FAILED"}, mode = EnumSource.Mode.EXCLUDE)
```

Comment:
Shouldn't we take maintenance mode out from here?

Reply:
No, because MAINTENENCE_MODE_ENABLED is part of the available statuses in com.sequenceiq.cloudbreak.api.endpoint.v4.common.Status and that status is used in core, unlike in Redbeams where it's not used and could be removed.